Source/WebCore/html/canvas/WebGLRenderbuffer.h

```diff
@@ -40,7 +40,6 @@ class WebGLRenderbuffer final : public WebGLObject {
     void setInternalFormat(GCGLenum internalformat)
     {
         m_internalFormat = internalformat;
-        m_initialized = false;
```

Comment:
Is this a "poor man's `std::optional`"? (ie. was `m_initialized` referring to whether `m_internalFormat` has been set? Should `m_internalFormat` be a `std::optional<GCGLenum>`?

Reply:
No, the code was unused and being removed here, not added. I'm unsure what it was originally for.